overlay/usr/lib/inithooks/bin/openvpn.py

```diff
@@ -85,6 +87,21 @@ def main():
 
     dialog = Dialog('TurnKey Linux - First boot configuration')
 
+    tun_exists = exists('/dev/net/tun')
+    if not tun_exists:
+        if is_interactive:
+            dialog.msgbox('Tun device not created', '''
+Failed to create `/dev/net/tun` device on boot, this is expected when running inside a non-privlidged container.
```

Comment:
Sorry mate, spotted a typo/spelling issue:

Should be `privileged` (not `privlidged`)